tpl/tplimpl/template_embedded.go

```diff
@@ -84,7 +84,7 @@ func (t *GoHTMLTemplate) EmbedTemplates() {
       <pubDate>{{ .Date.Format "Mon, 02 Jan 2006 15:04:05 -0700" | safeHTML }}</pubDate>
       {{ with .Site.Author.email }}<author>{{.}}{{ with $.Site.Author.name }} ({{.}}){{end}}</author>{{end}}
       <guid>{{ .Permalink }}</guid>
-      <description>{{ .Content | html }}</description>
+      <description>{{ .Summary | html }}</description>
```

Comment:
@rdwatters - that is a nuance of Hugo that I am not aware of.  I would go with whatever is considered best by PR reviewers.

All I know is that RSS feeds don't generally include entire articles in a feed, but rather a summary the norm.